www/templates/control/main-control.html

```diff
@@ -1,18 +1,32 @@
 <ion-view view-title="{{'control.profile' | i18next}}" ng-class="ion-view-background">
   <ion-content>
-    <div class="control-list-item">
+    
+    <!-- opcode and logout - this row has two clicks! (and an argument for one of them) -->
+    <!-- <div class="control-list-item">
       <div ng-click="copyToClipboard(settings.auth.opcode)" class="control-list-text"><pre>{{settings.auth.opcode}}</pre></div>
```

Comment:
`copyToClipboard()` is not super important functionality that users depend on (honestly, I really only added it to make my own life easier), so if it doesn't make sense to include it I don't think anyone will care.

That said, it might be nice anyway to support actions on the settings labels as well as on the icons. I'll leave it up to you

Reply:
I did end up eliminating copy to clipboard, since it was the only row that had multiple functions